mwviews/page_view_count.py

```diff
@@ -12,7 +12,18 @@ def __init__(self, project, page_name, views, bytes_returned):
 
     @classmethod
     def from_line(cls, line):
-        project, page_name, views, bytes_returned = line.strip().split(" ")
+        line_split_on_spaces = line.strip().split(" ")
```

Comment:
Would be nice to see the comment from the commit here as a code comment.  That way this will make sense when someone reads it in the future.